bindgen/ir/types/Type.cpp

```diff
@@ -0,0 +1,8 @@
+#include "Type.h"
+#include "../../Utils.h"
+
+std::string Type::str() const { return handleReservedWords(_str()); }
```

Comment:
Won't this call `handleReservedWords()` on stuff like `native.CFunctionPtr[...]`? It seems that it over-do it. How hard would it be to selectively call it for any type that returns a simple `name` and then get rid of `_str()`?

Reply:
It shouldn't be a problem